doc/page-cmakefunctions.h

```diff
@@ -0,0 +1,126 @@
+                     [HEADERS_GLOB [Globbing expressions]...]
+                     [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                     [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                     [HEADERS_NO_INSTALL hdrn1 [hdrn2...]]
+                     [HEADERS_GLOB_NO_INSTALL [Globbing expressions]...]
+                     [HEADERS_GLOB_RECURSE_NO_INSTALL [Globbing expressions]...]
+                     [LINK_LIBRARIES lib1 [lib2...]]
+                     [NO_EXPORT]
+                     [NO_INSTALL])
+\endcode
+
+This function adds a library with the name \e TargetName from the specified source and header files.
+
+\code
+facelift_add_executable(<TargetName>
+                        [SOURCES src1 [src2...]]
+                        [HEADERS hdr1 [hdr2...]]
+                        [SOURCES_GLOB gsrc1 [Globbing expressions]...]
+                        [HEADERS_GLOB ghdr1 [Globbing expressions]...]
+                        [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                        [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                        [LINK_LIBRARIES lib1 [lib2...]])
+
+\endcode
+
+This function adds an executable with the name \e TargetName from the specified source and header files.
+
+\code
+facelift_add_test(<TargetName>
+                  [SOURCES src1 [src2...]]
+                  [HEADERS hdr1 [hdr2...]]
+                  [SOURCES_GLOB [Globbing expressions]...]
+                  [HEADERS_GLOB [Globbing expressions]...]
+                  [SOURCES_GLOB_RECURSE [Globbing expressions]...]
+                  [HEADERS_GLOB_RECURSE [Globbing expressions]...]
+                  [LINK_LIBRARIES lib1 [lib2...]])
+\endcode
+
+This function adds a test with the name \e TargetName.
+
+
+\param TargetName The name of the target that will be created
+\param SOURCES List of source files used to build this target
+\param HEADERS List of header files used to build this target
+\param HEADERS_GLOB List of header files used to build this target provided as a wildcard expression
+\param SOURCES_GLOB List of source files used to build this target provided as a wildcard expression
+\param SOURCES_GLOB_RECURSE List of source files searched recursively inside folder and subfolders, used to build this target provided as a wildcard expression
+\param HEADERS_GLOB_RECURSE List of header files searched recursively inside folder and subfolders, used to build this target provided as a wildcard expression
+\param LINK_LIBRARIES List of libraries that will be linked with this target
+\param STATIC Force building a static library. If neither STATIC nor SHARED is given, a shared library will be created, if the global variable BUILD_SHARED_LIBS is ON, otherwise a static library is created
+\param SHARED Force building a shared library. If neither STATIC nor SHARED is given, a shared library will be created, if the global variable BUILD_SHARED_LIBS is ON, otherwise a static library is created
```

Comment:
I would rather remove "If neither STATIC nor SHARED is given, a shared library will be created, if the global variable BUILD_SHARED_LIBS is ON, otherwise a static library is created", since this is standard CMake stuff.
It should also not be repeated as it is now